### PR TITLE
Pull Request 2024-12-07 Language switch style change, Add warning modal on smart hightlight under motif

### PR DESCRIPTION
**Pull request by stoneYuan3**
image
image

### CODE REVIEW

File: src/components/Modals/SmartHighlight.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          onFocus={() => setModalOpen(true)}
+          onBlur={() => setModalOpen(false)}
+          className="w-full max-w-142.5 rounded-lg bg-white px-8 py-12 text-center dark:bg-boxdark md:px-17.5 md:py-15"
+        >
+          <h3 className="pb-2 text-xl font-bold text-black dark:text-white sm:text-2xl">
+            Warning!
+          </h3>
+          <span className="mx-auto mb-6 inline-block h-1 w-22.5 rounded bg-primary"></span>
+          <p className="mb-10">
+            Using smart highlight will overwrite your custom styles on related words.
```
Comment (teamsela) [inline src/components/Modals/SmartHighlight.tsx:101]: It should be "identical words", not related words

File: src/components/Modals/SmartHighlight.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        }`}
+      >
+        <div
+          ref={modal}
+          onFocus={() => setModalOpen(true)}
+          onBlur={() => setModalOpen(false)}
+          className="w-full max-w-142.5 rounded-lg bg-white px-8 py-12 text-center dark:bg-boxdark md:px-17.5 md:py-15"
+        >
+          <h3 className="pb-2 text-xl font-bold text-black dark:text-white sm:text-2xl">
+            Warning!
```
Comment (teamsela) [inline src/components/Modals/SmartHighlight.tsx:97]: Can you replace this with a warning icon that is used in delete study?

**Review by teamsela — CHANGES_REQUESTED** [review]
Changes required. Use the warning icon instead
Screenshot 2024-12-07 at 7 43 27 PM

File: src/components/Modals/SmartHighlight.tsx (hunk 2 above)
Comment (stoneYuan3) [reply src/components/Modals/SmartHighlight.tsx]: changed to something like this
image

File: src/components/Modals/SmartHighlight.tsx (hunk 1 above)
Comment (stoneYuan3) [reply src/components/Modals/SmartHighlight.tsx]: changed

**Pull request merged**